rescript/_utilities.py

```diff
@@ -107,9 +107,19 @@ def _read_dna_fasta(path):
     return skbio.read(path, format='fasta', constructor=skbio.DNA)
 
 
+def _read_dna_alignment_fasta(path):
+    return skbio.TabularMSA.read(path, format='fasta', constructor=skbio.DNA)
```

Comment:
I think we should avoid using `skbio.TabularMSA`. This basically reads the entire alignment into memory, which is what we are trying to avoid. I had to kill the process on my laptop.

Reply:
Oh, I see, this is just for the test cases. So we should be are okay there.